examples/knative/templates/Dockerfile.pcli

```diff
@@ -1,8 +1,25 @@
 FROM us.gcr.io/daisy-284300/veba/ce-ps-base:1.5
 
-ARG POWERCLI_VERSION="12.4.0.18633274"
+FROM lamw/ce-ps-base:1.5
+
+ARG POWERCLI_VERSION="13.1.0.21624340"
 
 RUN pwsh -c "\$ProgressPreference = \"SilentlyContinue\"; Install-Module VMware.PowerCLI -RequiredVersion ${POWERCLI_VERSION}" && \
     pwsh -c 'Set-PowerCLIConfiguration -ParticipateInCEIP $true -confirm:$false'
 
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends wget build-essential libssl-dev libncurses5-dev libsqlite3-dev libreadline-dev libtk8.6 libgdm-dev libdb4o-cil-dev libpcap-dev && \
+    cd /usr/src && \
+    wget https://www.python.org/ftp/python/3.7.9/Python-3.7.9.tgz && \
```

Comment:
nit: it'd be great if we could use the latest Python version here.

Reply:
This is currently a hard requirement from PCLI team on using this exact version as there's no guarantees for newer version, which has had issues.

In general, I would agree!